packages/@uppy/companion/src/server/emitter/redis-emitter.js

```diff
@@ -1,7 +1,13 @@
 const { EventEmitter } = require('node:events')
+const { default: safeStringify } = require('fast-safe-stringify')
 
 const logger = require('../logger')
 
+function replacer(key, value) {
```

Comment:
better to put replacer inside the function call so typescript can automatically infer the types for key and value, but we can also do that later when refactoring to typescript